project.mml

```diff
@@ -1616,7 +1618,7 @@ Layer:
           name,
           layer,
           COALESCE(
-           'highway_' || CASE WHEN tags @> 'ford=>yes' OR tags @> 'ford=>stepping_stones' THEN 'ford' ELSE NULL END,
+           'highway_' || CASE WHEN tags @> 'ford=>yes' OR tags @> 'ford=>stepping_stones' THEN 'ford' END,
            'leisure_' || CASE WHEN leisure IN ('slipway', 'track') THEN leisure ELSE NULL END,
```

Comment:
These two lines still have "ELSE NULL"

Reply:
My apologies, I should have better check, it's now fixed